.zuul/playbooks/run.yaml

```diff
@@ -0,0 +1,44 @@
+- hosts: all
+  become: yes
+  roles:
+  - role: config-golang
+  - role: export-cloud-openrc
+    vars:
+      cloud_name: 'citynetwork'
+  tasks:
+  - name: Run kubernetes E2E conformance tests with cloud-provider-openstack
```

Comment:
cloud-provider-openstack need switch to CAPO?

Reply:
As above. Please wait with the review. I'll need a lot of trial and error before it's ready for review

Reply:
But thx. I'll fix that :)